.NET/Microsoft.Recognizers.Text.NumberWithUnit/Extractors/INumberWithUnitExtractorConfiguration.cs

```diff
@@ -26,5 +26,7 @@ public interface INumberWithUnitExtractorConfiguration
         string ConnectorToken { get; }
 
         Regex CompoundUnitConnectorRegex { get; }
+
+        Regex SpecialTimeRegex { get; }

```

Comment:
The name here should be more descriptive that this is to filter potentially ambiguous cases. Not some special case of time extraction.